src/Agent.Listener/CommandLine/ConfigureAgent.cs

```diff
@@ -60,6 +60,9 @@ public class ConfigureAgent : ConfigureOrRemoveBase
         [Option(Constants.Agent.CommandLine.Flags.DisableLogUploads)]
         public bool DisableLogUploads { get; set; }
 
+        [Option(Constants.Agent.CommandLine.Flags.ReStreamLogsToFiles)]
```

Comment:
(not for this line) As this is a contribution, let's test thoroughly before merging

Reply:
I tested it with 3 variants:
1. --ReStreamLogsToFiles flag passed (should stream logs to server and store locally)
2. --DisableLogUploads flag passed (should store logs locally only)
3. no flags passed (should stream logs to server only

In all cases, it works as expected